vllm/entrypoints/openai/protocol.py

```diff
@@ -77,7 +77,7 @@ class CompletionRequest(BaseModel):
     # a string, array of strings, array of tokens, or array of token arrays
     prompt: Union[List[int], List[List[int]], str, List[str]]
     suffix: Optional[str] = None
-    max_tokens: Optional[int] = 16
+    max_tokens: Optional[int] = None
```

Comment:
One last commenti: for compatibility with the [OpenAI's completion API](https://platform.openai.com/docs/api-reference/completions/create#max_tokens) shouldn't the max number of tokens be left as it was before (16)?

Reply:
The doc says `max_tokens` is a integer or null variable and defaults to inf.
![image](https://github.com/vllm-project/vllm/assets/35923278/83e5a5ba-9c57-497b-b85e-01b3b0c67835)
Did I miss something?

Reply:
Ah, I got it. The behavior of `ChatCompletion` and `Completion` are not the same. I will fix it.

I apologize for my carelessness. Maybe I need to have a rest😭.